modules/RTC/adapter.screenshare.js

```diff
@@ -4013,21 +4013,24 @@ if ( (navigator.mozGetUserMedia ||
     }
 
     var downloadLink = AdapterJS.WebRTCPlugin.pluginInfo.downloadLink;
-    if(downloadLink) { // if download link
+
```

Comment:
We should try to not modify this file. On update we used to copy it directly from the Temasys repo and add this line:
https://github.com/BeatC/lib-jitsi-meet/blob/35636308c361485b0d6b003fef810ae9e6f6bedc/modules/RTC/adapter.screenshare.js#L2

If we make changes it may be hard to figure out.